tests/sqllogictests/suites/query/functions/02_0002_function_others.test

```diff
@@ -67,3 +67,7 @@ SELECT array(1);
 ----
 [1]
 
+query III
+SELECT telnet('localhost', 3307) = 'Ok', telnet('bad_localhost', 3307) != 'Ok', telnet('google.com', 80) != 'Ok'
```

Comment:
this might have risks for untrusted users to probe the available ports or side attack.